provision/ubuntu/install.sh

```diff
@@ -132,6 +132,10 @@ git clone --depth=1 ${LIBBPF_GIT}
 cd /tmp/libbpf/src
 make -j "$(getconf _NPROCESSORS_ONLN)"
 PREFIX="/usr" sudo make install
```

Comment:
Isn't that what `PREFIX` here is for?

Reply:
https://github.com/cilium/libbpf/blob/master/src/Makefile#L78

Reply:
Do you mean it's the LIBSUBDIR we'd need to change?